projects/storefrontlib/src/lib/storefront.module.ts

```diff
@@ -12,6 +16,7 @@ import { StorefrontModuleConfig } from './storefront-config';
 
 @NgModule({
   imports: [
+    ConfigurableRoutesModule,
```

Comment:
What about we import this module inside the RoutingModule?

Reply:
Good point. Thanks.